ui/DesignSystem/Component/ModalOverlay.svelte

```diff
@@ -38,9 +39,14 @@
   }
 </style>
 
-<div class="modal" class:hide={!$store.show} data-cy="modal">
+<div class="modal" class:hide={store === null} data-cy="modal">
   <div class="overlay" on:click={clickOutside} />
   <div class="content">
-    <svelte:component this={modalRoutes[$store.route]} on:hide={modal.hide} />
+    {#if store !== null}
+      <svelte:component
+        this={store.modal}
+        on:hide={store.onHide}
```

Comment:
Since we’re not using the event dispatchers we can remove `on:hide`

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1814/commits/a9a08d9b94e20529f72c04fc1a118b6dd2365e4c